plugins/cuda/cuda_plugin.c

```diff
@@ -481,10 +497,12 @@ int cuda_plugin_init(int stage)
 		}
 	}
 
-	if (!fault_injected(FI_PLUGIN_CUDA_FORCE_ENABLE) && access("/dev/nvidiactl", F_OK)) {
-		pr_info("/dev/nvidiactl doesn't exist. The CUDA plugin is disabled.\n");
-		plugin_disabled = true;
-		return 0;
+	if (!fault_injected(FI_PLUGIN_CUDA_FORCE_ENABLE)) {
```

Comment:
nit: !fault_injected() && !has_gpu()

Reply:
Thanks, I also noticed that (if I understand correctly), setting `FI_PLUGIN_CUDA_FORCE_ENABLE` should skip the GPU device check. In other words, the condition should be:
```
!fault_injected(FI_PLUGIN_CUDA_FORCE_ENABLE) || !has_nvidia_gpu()
```
I've also updated the zdtm test to run with `--criu-plugin cuda` to test this change.